### PR TITLE
VR Controls

### PR DESCRIPTION
**Pull request by kyle-emmerich**

### CODE REVIEW

File: PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl.rbxmx [hunk 1]
```diff
@@ -68,11 +66,11 @@ function MasterControl:Init()
 			end
 			
 			local adjustedMoveValue = moveValue
-			if not NoVR and UserInputService.IsVREnabled then
-				local inverseHeadCFrame = UserInputService.UserHeadCFrame:inverse()
-				local yawOnlyLookVector = Vector3.new(inverseHeadCFrame.lookVector.X, 0, inverseHeadCFrame.lookVector.Z).unit
-				local yawOnlyCFrame = CFrame.new(Vector3.new(0,0,0), yawOnlyLookVector)
-				adjustedMoveValue = yawOnlyCFrame * adjustedMoveValue
+			if HasVRAPI and UserInputService.VREnabled and workspace.CurrentCamera.HeadLocked then
```
Comment (hockeychaos) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl.rbxmx]: So HeadLocked will determine whether the character moves in the direction you are looking?

Comment (kyle-emmerich) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl.rbxmx]: HeadLocked is true by default and means that VR offset is being applied by the engine.

File: PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/AttachCamera.rbxmx [hunk 2]
```diff
@@ -66,8 +66,8 @@ local function CreateAttachCamera()
 
 
 			camera.Focus = CFrame.new(subjectPosition)
-			module.cframe = CFrame.new(camera.Focus.p - (zoom * newLookVector), camera.Focus.p)
-			self.LastCameraTransform = module.cframe
+			camera.CFrame = CFrame.new(camera.Focus.p - (zoom * newLookVector), camera.Focus.p)
+			self.LastCameraTransform = camera.CFrame
```
Comment (SolarCrane) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/AttachCamera.rbxmx]: Is this code tested yet? I don't recall Camera ever having a `CFrame` property, you might want to change it to `CoordinateFrame`

Comment (kyle-emmerich) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/AttachCamera.rbxmx]: It does. I'll figure out which release it was added and make sure it's present everywhere.

File: PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/FollowCamera.rbxmx [hunk 3]
```diff
@@ -151,10 +149,13 @@ local function CreateFollowCamera()
 			end
 			local newLookVector = self:RotateCamera(self:GetCameraLook(), self.RotateInput)
 			self.RotateInput = Vector2.new()
+
+			local height = self:IsInFirstPerson() and 0 or self.currentHeight
```
Comment (SolarCrane) [inline PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/FollowCamera.rbxmx]: I see this exact line has been added in multiple camera files, can we extract it into a function instead?

**Pull request merged**